views/SprinklerView/TurnOnFanSelect/TurnOnSprinklerSelect.tsx

```diff
@@ -0,0 +1,148 @@
+import React, { FC, useMemo } from 'react';
+import {
+  Button,
+  CheckBox,
+  Input,
+  Select,
+  Typography,
+  useCheckBoxValue,
+  useInputValue,
+  useSelectValue,
+} from '../../../.storybook/stories';
+import { selectDevicesWithType } from '../../../redux/devicesSlice';
+import { useAppSelector } from '../../../redux/hooks';
+import { DeviceType, SprinklerSchedule } from '../../../types';
+import { sendAPIRequest } from '../../../utils';
+
+type TurnOnSprinklerSelectProps = {
+  sprinklerBaseParams: any;
```

Comment:
Set proper type to fanBaseParams like TempSensorBaseParams in [TempSensorView.tsx](https://github.com/inzynierka-home-iot/frontend-rncli/pull/21/files#diff-df6eb1cf97e6dff09228f26e4c6502f320c9454dd41bd61477c99a5f9e95cac8)